### PR TITLE
update to Remix Commands

### PR DESCRIPTION
**Pull request by ryestew**
fix https://github.com/ethereum/remix-project/issues/88

### CODE REVIEW

File: docs/remix_commands.md [hunk 1]
```diff
@@ -28,66 +25,22 @@ In the console, you can run the commands listed below.  Once you start to type a
 
 **remix.loadurl(url)**: Load the given url in the file explorer. The url can be of type github, swarm or ipfs.
 
-**remix.setFile(path, content)**: set the content of the file located at the given path
```
Comment (Aniket-Engg) [inline docs/remix_commands.md:31]: Is this command not working for remix anymore?

File: docs/remix_commands.md [hunk 2]
```diff
@@ -28,66 +25,22 @@ In the console, you can run the commands listed below.  Once you start to type a
...
-
-**remix.setproviderurl(url)**: Change the current provider to Web3 provider and set the url endpoint.
```
Comment (Aniket-Engg) [inline docs/remix_commands.md:33]: Same for this one

File: docs/remix_commands.md [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
+**remix**:  Remix has a number of CLI commands for loading & executing file in a workspace.  See the list below.
  
-+ *web3*: The [web3.js](https://web3js.readthedocs.io/en/1.0/) library is a collection of modules which contain specific functionality for the ethereum ecosystem.
+**ethers**: Remix IDE enables the use of ethersjs commands. See the [Ethers docs](https://docs.ethers.io/) for the full list.
 
-+ *swarmgw*: This library can be used to upload/download files to Swarm via https://swarm-gateways.net/.
+**web3**: Remix IDE enable the use of web3js commands. See the [Web3js docs](https://web3js.readthedocs.io/) for the full list.
 
-#### Here's the list of commands
-**remix.debug(hash)**: Start debugging a transaction.
```
Comment (Aniket-Engg) [inline docs/remix_commands.md:15]: and this too

Comment (ryestew) [reply docs/remix_commands.md:15]: Putting in:
remix.debug('0x0e47a0e996f14bfd5db3d15e0e06b5dd07c4c248dab12cd352b29ae3e7c67471')
gives:
remix.debug is not a function

File: docs/remix_commands.md (hunk 2 above)
Comment (ryestew) [reply docs/remix_commands.md:33]: Input: remix.setproviderurl('HTTP://127.0.0.1:7545')
returns: remix.setproviderurl is not a function

File: docs/remix_commands.md (hunk 1 above)
Comment (ryestew) [reply docs/remix_commands.md:31]: input: remix.setFile("./contracts/f", "fred")
returns: remix.setFile is not a function

Comment (Aniket-Engg) [reply docs/remix_commands.md:31]: ok

File: docs/remix_commands.md (hunk 2 above)
Comment (Aniket-Engg) [reply docs/remix_commands.md:33]: ok

File: docs/remix_commands.md (hunk 3 above)
Comment (Aniket-Engg) [reply docs/remix_commands.md:15]: ok

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**